src/General/ProfilePicture/ProfilePicture.tsx

```diff
@@ -9,62 +9,51 @@ import {
   ProfilePictureContent,
 } from '../../Style/General/ProfilePictureStyle';
 
-class ProfilePicture extends React.Component<Props, State> {
-  state = {
-    isHover: false,
-  };
-
-  handleMouseEnter = () => {
-    const { editable } = this.props;
-
+const ProfilePicture = ({
+  editable,
+  children,
+  className,
+  ...defaultProps
+}: Props) => {
+  const [isHover, setIsHover] = React.useState<boolean>(false);
+
+  const handleMouseEnter = React.useCallback(() => {
```

Comment:
I think we shouldn't use `useCallback` here because it is passed to a `div` rather than a component.

Reply:
Changed to regular function.